lib/transform/classic.js

```diff
@@ -125,6 +127,9 @@ module.exports = function transformClassicComponent(root, options) {
   let classNameBindings = findClassNameBindings(properties);
   debug('classNameBindings: %o', classNameBindings);
 
+  let dataTestAttributes = findDataTestAttributes(properties);
+  debug('dataTestAttributes: %o', dataTestAttributes);
```

Comment:
hmm, instead of creating a dedicated thing here, I'm wondering if it would be easier to just add all the `data-test-*` properties to the attribute bindings? that's what ember-test-selectors does internally and it would also support computed properties that way. I'm just not sure if `data-test-foo={{this.data-test-foo}}` is valid code due to the `-` in the property name 🤔